### ISSUE TITLE
IP forwarding

### ISSUE THREAD

**Issue by daschmidt1994** [hassio-addons/addon-wireguard#190]
# Problem/Motivation

A device from the local network can't ping to a client from the vpn. 
My usecase: The Homeassistant run on a Raspberry Pi 4(192.168.178.19) with the wireguard addon. In this network is also 
my synology ds920+(192.168.178.200). At the house of my sister there is a raspberry(10.6.0.3) connected per vpn with my network for backup.
But the ds920+ can't connect the raspberry by my sister. In my router the routing is activated.  

## Steps to reproduce

Ping from a local device a client from the vpn

## Proposed changes
I think its the ipv4 forward the problem. 
If I trie to use the commands from the documentation to forward ipv4. 
There I get this error: sysctl: error setting key 'net.ipv4.ip_forward': Read-only file system

"IP forwarding is disabled on the host system!"

IP forwarding is the ability for an operating system to accept incoming network packets on one interface, recognize that it is not meant for the system itself, but that it should be passed on to another network, and then forwards it accordingly.

Basically, it allows for data coming in from your VPN client will be routed to other places like your home network, or the internet.

To enable IP forwarding, run the following commands directly on your host system (e.g., Ubuntu, Debian).

sudo sysctl -w net.ipv4.ip_forward=1
echo "net.ipv4.ip_forward = 1" | sudo tee -a /etc/sysctl.conf
sudo sysctl -p /etc/sysctl.conf
